fendermint/app/options/src/genesis.rs

```diff
@@ -36,6 +40,26 @@ pub enum GenesisCommands {
     IntoTendermint(GenesisIntoTendermintArgs),
 }
 
+#[derive(Args, Debug)]
+pub struct GenesisSetEAMPermissionsArgs {
+    #[arg(
+        long,
+        short,
+        default_value = "unrestricted",
+        help = "Permission mode (unrestricted/allowlist) that controls who can deploy contracts in the subnet"
+    )]
+    pub mode: String,
+
+    #[arg(
+        long,
+        short,
+        value_delimiter = ',',
+        value_parser = parse_signer_addr,
+        help = "List of addresses that can deploy contract. Field is ignored if mode is unrestricted"
+    )]
+    pub addresses: Vec<SignerAddr>,
```

Comment:
Is it okay for these to be FVM addresses? It might let in things that aren't necessarily Ethereum compatible.

Reply:
It awards more flexibility (e.g. in case you want a native multisig, f1, or f3 to act as a deployer). We could have an intelligent type that recognizes Eth addresses and converts them to f410.